server/service/src/preference/preferences/show_contact_tracing.rs

```diff
@@ -8,4 +8,12 @@ impl Preference for ShowContactTracing {
     fn key(&self) -> &'static str {
         "show_contact_tracing"
```

Comment:
I did consider, should key be an enum? Would have to create a migration for each new pref, but would maybe feel a bit cleaner than string, especially when referenced on frontend?